docs/persistence/bigquery.md

```diff
@@ -0,0 +1,53 @@
+# Bigquery Datasets
+
+!!! info
+    This feature is only available in GCP clusters.
+
+## NAIS Application yaml manifest options
+Full documentation of all available options can be found over at: [`spec.gcp.bigQueryDatasets[]`](../../nais-application/application#gcpbigquerydatasets).
+
+Example of an application using a `nais.yaml` provisioned BigQuery Dataset can be found here: [testapp](https://github.com/nais/testapp/blob/master/pkg/bigquery/bigquery.go).
+
+## Minimal Working Example
+=== "You can request a MWE Google Cloud BiqQuery Dataset through the NAIS Application manifest like so:"
```

Comment:
Hva mener dere med `You can request a MWE`?

Reply:
Du kan be om å få opp en minimal working example